framework/config/src/main/java/org/apache/cloudstack/framework/config/ConfigKey.java

```diff
@@ -229,6 +229,10 @@ public ConfigKey(String category, Class<T> type, String name, String defaultValu
         this(type, name, category, defaultValue, description, isDynamic, Scope.Global, null);
     }
 
+    public ConfigKey(String category, Class<T> type, String name, String defaultValue, String description, Scope scope, boolean isDynamic) {
+        this(type, name, category, defaultValue, description, isDynamic, scope, null);
+    }
```

Comment:
is there really need for a new constructor? We can call `public ConfigKey(Class<T> type, String name, String category, String defaultValue, String description, boolean isDynamic, Scope scope, T multiplier)` with `null` as last parameter, can we?

This class now has 15 constructors, but only one that does any real constructing. (no real -1 but not really a fan either)